pkg/querier/block.go

```diff
@@ -178,24 +179,44 @@ func (it *blockQuerierSeriesIterator) At() (int64, float64) {
 	return t, v
 }
 
-func (it *blockQuerierSeriesIterator) Next() bool {
+// AtHistogram returns (0, nil) because histograms aren't implemented yet.
+func (it *blockQuerierSeriesIterator) AtHistogram() (int64, *histogram.Histogram) {
+	return 0, nil
+}
+
+// AtFloatHistogram returns (0, nil) because histograms aren't implemented yet.
+func (it *blockQuerierSeriesIterator) AtFloatHistogram() (int64, *histogram.FloatHistogram) {
+	return 0, nil
```

Comment:
same comment about returning errors

Reply:
Returning error was a wrong suggestion. You can panic here for now like you did in some other place.